bridge/endpointActions/GetProbingInfo.cpp

```diff
@@ -0,0 +1,45 @@
+#include "ApiActions.h"
+#include "api/ConferenceEndpoint.h"
+#include "bridge/Mixer.h"
+#include "bridge/MixerManager.h"
+#include "bridge/RequestLogger.h"
+#include "nlohmann/json.hpp"
+#include "transport/ProbeServer.h"
+
+namespace bridge
+{
+httpd::Response getProbingInfo(ActionContext* context,
+    RequestLogger& requestLogger,
+    const httpd::Request&,
+    const utils::StringTokenizer::Token&)
+{
+    nlohmann::json responseBodyJson;
+    nlohmann::json iceJson;
+
+    auto& probeServer = context->probeServer;
+
+    const auto credentials = probeServer.getProbeCredentials();
+
+    iceJson["ufrag"] = credentials.first;
+    iceJson["pwd"] = credentials.second;
+
+    nlohmann::json candidatesJson = nlohmann::json::array();
+
+    for (auto candidate : probeServer.getProbeCandidates())
+    {
+        nlohmann::json candidateJson;
+        candidateJson["protocol"] = candidate.transport == ice::TransportType::TCP ? "tcp" : "udp";
+        candidateJson["ip"] = candidate.address.ipToString();
+        candidateJson["port"] = candidate.address.getPort();
+        candidatesJson.push_back(candidateJson);
```

Comment:
As @olofkallander recommended I left only those properties required for RTT measurement. The "candidates" won't be used for real ICE.

Reply:
Well that's true. But not sure if we have to generate that information anyway to generate the sdp. Then if can provide, it would be easier to generate an sdp. And if you have multiple endpoints with private IPs and public IPs at least the ICE type would be useful. @olofkallander, wdyt?

Reply:
Is there a point of having separate probing candidates at all then? We could build list of regular ICE candidates for all the TCP and UDP ports on startup for chosen (generated) ICE credentials; and have TransportFactory to expose them to API.

Reply:
I changed it to report all the properties we report for ICE candidates in transport description.